### DISCUSSION TITLE
Allow casters besides mage/warlock/priest to roll for cloth gear in RDF.

### DISCUSSION THREAD

**Discussion by Karponn** [chromiecraft/chromiecraft#3406] [Suggestions]
Due to the itemization choices blizzard made in classic, the blizzlike RDF loot rules don't serve the community as well as they should.
It's frustrating to try to get geared up doing random dungeons when you can't roll for your BiS gear. Us non-priest healers and other leather/mail wearing casters deserve an equal shot at the gear in RDF. Often people will be nice enough to hand over the item if no one needed it but what makes the problem even more annoying is that just as often people will leave the dungeon before you have a chance to even ask or they just don't respond. It also feels bad to ask to roll on an item someone already won so I haven't done that.
I'm not familiar enough with physical DPS but it might be worth it to take a look at shaman/hunter/warrior/paladin being allowed to roll for leather as well.
I'm not sure of the difficulty of implementing this as suggested, but I think it would be fine to let everyone roll for gear they can wear as well. Afaik it's ninjaing this is trying to prevent, but personally I haven't seen ninjaing unless it's a new player who doesn't know any better but the frustration this brings is there frequently.

Thanks,

Karpon

**Comment by Santaria94** [comment]
What you're suggesting is actually Blizzlike and that's how it was implemented on AzerothCore, but for some reason Chromiecraft changed the rules for high level classic dungeons. They are convinced that what they did is right, and attempts to explain something about the specifics and imbalance of the classic loot tables are usually unsuccessful.

https://github.com/azerothcore/azerothcore-wotlk/issues/7467
